docs/source/gbd2019_models/risk_exposures/stunting/index.rst

```diff
@@ -0,0 +1,337 @@
+
++----------------------------------------------+
+| Stunting category definition (range -7 to +7)|
++=======+======================================+
+| TMREL |  >= -1                               |            
++-------+--------------------------------------+
+| Mild  |  < -1 to -2 Z score                  |
++-------+--------------------------------------+
+| Mod   |  < -2 to -3 Z score                  |
++-------+--------------------------------------+
+| Sev   |  < -3 Z score                        |
++-------+--------------------------------------+
+
+.. _stunting1.1.2:
+
+Input data
+----------
+
+Two types of input data are used in CGF estimation:  
+
+  1. **Tabulated report data**. This data does not report individual anthropometric measurements. It only reports the prevalence of forms of CGF in a sample size. For example, this data would may report a 15% prevalence of moderate stunting out of a nationally representative sample of 5,000 children.
+
+  2. **Microdata**. This data does have individual anthropometric measurements. From these datasources, GBD can see entire distributions of CGF, while also collapsing them down to point prevalences like moderate and severe CGF. 
+
+
+.. _stunting1.1.3:
+
+Exposure estimation
+------------------- 
+
+In modeling CGF, all data types go into ST-GPR modeling. We have STGPR models for moderate, severe, and mean stunting, wasting, and underweight. The output of these STGPR models is an estimate of moderate, severe, and mean stunting, wasting, and underweight for all under 5 age groups, all locations, both sexes, and all years. 
+
+We also take the microdata sources and fit ensemble distributions to the shapes of the stunting, wasting, and underweight distributions. By doing this we can find characteristic shapes of stunting, wasting, and underweight curves. Once we have ST-GPR output as well as weights that define characteristic curve shapes, the last step is to combine them. We anchor the curves at the mean output from ST-GPR, use the curve shape from the ensemble distribution modeling, and then use an optimization function to find the standard deviation value that allows us to stretch/shrink the curve to best match the moderate and severe CGF estimates we got from ST-GPR. The final CGF estimates are the area under the curve for this optimized curve.
+
+
+
+.. note::
+  
+  Note that the z-score ranges from -7 to +7. If we limit ourselves to Z-scores between -4 and +4, we will be excluding a lot of kids.
+
+  In the paper that Ryan (GBD modeller for CGF and LWBSG) is working on right now, he presents the first results ever for "extreme" stunting which we define as kids with stunting Z scores below -4. For Ethiopia, that's about 7% of kids. So it's non-trivial!
```

Comment:
This makes me think we should include the extreme stunting category in our model, as well.

Reply:
for documentation clarity: maybe add Ryan's last name, and date for "now" (Ryan Fitzgerald / june 2021)